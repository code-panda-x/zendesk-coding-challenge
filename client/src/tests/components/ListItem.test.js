import React from 'react';
import Enzyme, { shallow } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
import toJSON from 'enzyme-to-json'
import TicketListItem from '../../Tickets/ListItem';
import ticketData from '../fixtures/ticketData'

Enzyme.configure({
  adapter: new Adapter()
})

test('Should render TicketListItem correctly', () => {
  const wrapper = shallow(<TicketListItem key={ticketData.id} ticket={ticketData} />)
  expect(toJSON(wrapper)).toMatchSnapshot()
})
