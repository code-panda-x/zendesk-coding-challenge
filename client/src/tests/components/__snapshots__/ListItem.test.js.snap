// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render TicketListItem correctly 1`] = `
<div
  className="ticket-list-item"
  onClick={[Function]}
>
  <div
    className="ticket-list-item__header"
  >
    <p>
      <span
        className="subject"
      >
        New ticket
      </span>
      <span
        className="right"
      >
         ID: 
        1
      </span>
    </p>
  </div>
  <div
    className="ticket-list-item__header"
  >
    <p>
      <span
        className="date"
      >
        Wed Nov 24 2021
      </span>
      <span
        className="right"
      >
         Status: 
        open
      </span>
    </p>
  </div>
</div>
`;
